src/Delegates/HandlesIndex.php

```diff
@@ -61,7 +61,7 @@ public function getOrCreateIndex(string $uid, array $options = []): Indexes
         try {
             $index = $this->getIndex($uid, $options);
         } catch (HTTPRequestException $e) {
-            if (\is_array($e->httpBody) && 'index_not_found' !== $e->httpBody['errorCode']) {
+            if (!\is_array($e->httpBody) || 'index_not_found' !== $e->httpBody['errorCode']) {
                 throw $e;
             }
             $index = $this->createIndex($uid, $options);
```

Comment:
```suggestion
            if (\is_array($e->httpBody) && 'index_not_found' == $e->httpBody['errorCode'])
            {
                $index = $this->createIndex($uid, $options);
            } else {
               throw $e;
            }
```

The first part of the if the condition was probably added to ensure the `httpBody` has content in it. As it could be an error not related to MeiliSearch in which case the httpBody might be empty.

I try to find a more bulletproof solution. What do you think of this suggestion?

Reply:
works for me!

Reply:
Could you commit the suggestion?

Reply:
Done